tests/test_results_table.py

```diff
@@ -0,0 +1,126 @@
+import os
+import tempfile
+from unittest import TestCase
+
+from hdmf.common import HDF5IO, get_manager, get_hdf5io, EnumData
+from hdmf_ml import ResultsTable
+
+import numpy as np
+
+
+class ResultsTableTest(TestCase):
```

Comment:
It would be nice to have round-trip tests here. I am happy to help with that if you prefer.